modules/core/include/opencv2/core/hal/intrin_legacy_ops.h

```diff
@@ -0,0 +1,111 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+// This file has been created for compatibility with older versions of Universal Intrinscs
+// Binary operators for vector types has been removed since version 4.11
+// Include this file manually after OpenCV headers if you need these operators
+
+#ifndef OPENCV_HAL_INTRIN_LEGACY_OPS_HPP
+#define OPENCV_HAL_INTRIN_LEGACY_OPS_HPP
+
+#ifdef __OPENCV_BUILD
+#error "Universal Intrinsics operators are deprecated and should not be used in OpenCV library"
+#endif
+
+#ifdef __riscv
+#warning "Operators might conflict with built-in functions on RISC-V platform"
+#endif
+
+#if defined(CV_VERSION) && CV_VERSION_MAJOR == 4 && CV_VERSION_MINOR < 11
+#warning "Older versions of OpenCV (<4.11) already have Universal Intrinscs operators"
+#endif
```

Comment:
It is only since OpenCV 4.11 the Universal Intrinscs operators have been removed?

I don't understand, I had to [fix some code](https://github.com/lagadic/visp/commit/9ed0300507e13dddd83fd62a799f5039025ea44e) since OpenCV 4.9 release.

https://github.com/opencv/opencv/pull/24371

Reply:
Thanks a lot for the feedback. The operator overload was removed due to problems with RISC-V platform. The header with overloaded operators was added for compatibility on previously supported platforms (we have such issue report). You do not need to touch your code. v_xxx functions is the way to go further.

Reply:
@s-trinh , you are right. Operators has been removed from AArch64 backend since 4.9 and from other backends since 4.11.